### ISSUE TITLE
Problems with Module and Commands

### ISSUE THREAD

**Issue by Frxhb** [ethanbergstrom/Cobalt#18]
Hey there,

I have some problems searching for packages using your PowerShell Module.

If I run `Find-WinGetPackage -ID xy` or `Get-WinGetPackage -ID xy` I receive the following error:

![2023-01-26 10_37_17-Greenshot](https://user-images.githubusercontent.com/68739220/214803832-fc47d211-39e8-47c2-abac-f660ff143318.png)

Yes, Firefox is installed. No, GIMP is not installed.

Windows-Paket-Manager v1.4.10173
Windows: Windows.Desktop v10.0.22621.1105
Systemarchitektur: X64
Paket: Microsoft.DesktopAppInstaller v1.19.10173.0

Please tell me if you need more information.
Thanks for your work.

english transaltion (used deepl):
```
Select-String : The string "{0} updates available." is not a valid regular expression: "{0} updates available." will be
parsed - quantifier {x,y} after nothing.
In C:\Program Files\WindowsPowerShell\Modules\Cobalt\0.3.2\Cobalt.psm1:637 characters:28
+ ...  ($output | Select-String -Pattern $languageData.AvailableUpgrades,'- ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo : InvalidArgument: (:) [Select-String], ArgumentException
    + FullyQualifiedErrorId : InvalidRegex,Microsoft.PowerShell.Commands.SelectStringCommand
```

**Comment by ethanbergstrom** [comment]
Thanks for including the english translation, that helped me track down the issue. 
It looks like Microsoft broke this by updating their string formats with microsoft/winget-cli#2783

**Comment by ethanbergstrom** [comment]
@Frxhb I'm not able to test this myself, but I _think_ this should be resolved for you by upgrading Cobalt to v0.3.3, which is out now on PSGallery.

If it's not, please let me know.

**Comment by Frxhb** [comment]
> @Frxhb I'm not able to test this myself, but I _think_ this should be resolved for you by upgrading Cobalt to v0.3.3, which is out now on PSGallery.
> 
> If it's not, please let me know.

Thanks alot. Now its working fine.
